install/install.sh

```diff
@@ -59,14 +59,23 @@ cleanup() {
 
 trap cleanup EXIT
 
+# Test if stdout is a terminal before showing progress
+if [[ ! -t 1 ]]; then
+  CURL_OPTIONS="--no-progress-meter"
+  WGET_OPTIONS="--no-verbose"
+else
+  CURL_OPTIONS="--progress-bar"
+  WGET_OPTIONS="--show-progress"
+fi
+
 if hash curl 2> /dev/null; then
-  HTTP_CODE="$(curl -SL --progress-bar "$DOWNLOAD_URL" --output "$TEMP_FILE" --write-out "%{http_code}")"
+  HTTP_CODE="$(curl -SL "$CURL_OPTIONS" "$DOWNLOAD_URL" --output "$TEMP_FILE" --write-out "%{http_code}")"
```

Comment:
You probably want to leave the quotes out here in case the `*_OPTIONS` vars above start gaining flags. Same below for wget.

Reply:
thanks changed that.